geonode_dominode/geonode_dominode/apps.py

```diff
@@ -26,6 +30,19 @@ def run_setup_hooks(*args, **kwargs):
     if celeryapp not in settings.INSTALLED_APPS:
         settings.INSTALLED_APPS += (celeryapp, )
```

Comment:
messing with INSTALLED_APPS here seems wrong. Can't this go into the standard `settings.py` instead?

Reply:
This is coming directly from `geonode-project`.
I haven't tried moving it, but I think this is because the celery app needs to be put after the settings is loaded to avoid circular dependency (they load the settings in the celeryapp to detect tasks).

Reply:
yuck :disappointed: 

I'm sure there is a proper way to do this... but anyway, if it comes from upstream lets leave it like that for now